.eslintrc.cjs

```diff
@@ -22,6 +28,7 @@ module.exports = {
     'import/extensions': 0, // don't require file ext in imports
     'import/no-extraneous-dependencies': ['error', { devDependencies: true }], // stop eslint complaining about deps vs devDeps
     'no-param-reassign': ['error', { props: false }], // Phaser requirement
+    'no-shadow': 'off',
```

Comment:
Do we have to completely turn this off for Enums? 
`"@typescript-eslint/no-shadow": "warn",` - I found this online and I _think_ it's pulling from a different set of defined rules? Tho I could be wrong and it's the same rule but changing it to a warning 🤔 

https://github.com/ScottyXIII/bigheaded/blob/enemies/.eslintrc.cjs#L26

Reply:
wait scrap that last comment, I think you're doing the same thing just on a global scale above 🤔 

`plugin:@typescript-eslint/eslint-recommended`